src/main/java/consul/ConsulChain.java

```diff
@@ -20,4 +25,30 @@ protected ConsulChain(Consul consul) {
     public Consul consul() {
         return consul;
     }
+
+    public static JsonNode checkResponse(HttpRequest request) throws ConsulException {
+         String body;
```

Comment:
nit: Body can be moved inside the try, or even removed completely.
{code}
return parseJson(response.getBody());
{code{